accounting/accounting/doctype/stock_entry_table_item/stock_entry_table_item.json

```diff
@@ -0,0 +1,66 @@
+{
+ "actions": [],
```

Comment:
There seem to be two table doctypes... remove one? 

also naming should be `Stock Entry Item` as per our naming conventions